### PR TITLE
[Form] Ability to clear form errors

### PR DESCRIPTION
**Pull request by colinodell**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (symfony/symfony#14233) |
| Applies to | 3.0 (master) |
| Fixed tickets |  |

### CODE REVIEW

File: components/form/introduction.rst [hunk 1]
```diff
@@ -697,6 +697,13 @@ method to access the list of errors. It returns a
     This is useful, for example, if you want to use PHP's ``array_`` function
     on the form errors.
 
+Clearing Form Errors
+~~~~~~~~~~~~~~~~~~~~
+
+Any errors can be manually cleared using the :method:`Symfony\\Component\\Form\\FormInterface::clearErrors`
+method. This is useful when you'd like to validate the form without showing validation errors to the user
+(i.e. during a partial AJAX submission or :ref:`dynamic form modification <cookbook-form-dynamic-form-modification>`).
```
Comment (colinodell) [inline components/form/introduction.rst]: Could anyone please provide some guidance on how I'd link to [this cookbook article](http://symfony.com/doc/current/cookbook/form/dynamic_form_modification.html)?  Apparently I've done it wrong :-/

Comment (xabbuh) [inline components/form/introduction.rst]: It looks like you want to do the following:

```rst
:doc:`dynamic form modification </cookbook/form/dynamic_form_modification>`
```

**Comment by colinodell** [comment]
Thanks for the help @xabbuh! I've corrected the link and the tests now pass.

**Comment by WouterJ** [comment]
Closing as upstream PR was rejected

**Pull request closed without merging**